retroshare-gui/src/main.cpp

```diff
@@ -233,28 +223,57 @@ feenableexcept(FE_INVALID | FE_DIVBYZERO);
-            >> parameter('o',"opmode"        ,conf.opModeStr      ,"opmode"    ,"Set Operating mode (Full, NoTurtle, Gaming, Minimal)."       ,false)
-            >> parameter('t',"opmode"        ,conf.userSuppliedTorExecutable,"tor"    ,"supply full tor eecutable path."       ,false);
+    as      >> option('s',"stderr"           ,conf.outStderr      ,"output to stderr instead of log file "    )
+            >> option('u',"udp"              ,conf.udpListenerOnly,"Only listen to UDP "                      )
+            >> parameter('c',"base-dir"      ,conf.optBaseDir     ,"directory", "Set base directory "                                         ,false)
+            >> parameter('l',"log-file"      ,conf.logfname       ,"logfile"   ,"Set Log filename "                                           ,false)
+            >> parameter('d',"debug-level"   ,conf.debugLevel     ,"level"     ,"Set debug level "                                            ,false)
+            >> parameter('i',"ip-address"    ,conf.forcedInetAddress,"nnn.nnn.nnn.nnn", "Force IP address "    ,false)
+            >> parameter('p',"port"          ,conf.forcedPort     ,"port"      ,"Set listenning port "                                 ,false)
+            >> parameter('o',"opmode"        ,conf.opModeStr      ,"opmode"    ,"Set mode (Full, NoTurtle, Gaming, Minimal) "       ,false)
+            >> parameter('t',"opmode"        ,conf.userSuppliedTorExecutable,"tor"    ,"supply full tor executable path "       ,false);
 #ifdef RS_JSONAPI
 	as      >> parameter('J', "jsonApiPort", conf.jsonApiPort, "jsonApiPort", "Enable JSON API on the specified port", false )
-	        >> parameter('P', "jsonApiBindAddress", conf.jsonApiBindAddress, "jsonApiBindAddress", "JSON API Bind Address.", false);
+            >> parameter('P', "jsonApiBindAddress", conf.jsonApiBindAddress, "jsonApiBindAddress", "JSON API Bind Address ", false);
 #endif // ifdef RS_JSONAPI
 
 #ifdef LOCALNET_TESTING
-	as      >> parameter('R',"restrict-port" ,portRestrictions             ,"port1-port2","Apply port restriction"                   ,false);
+    as      >> parameter('R',"restrict-port" ,portRestrictions             ,"port1-port2","Apply port restriction "                   ,false);
 #endif // ifdef LOCALNET_TESTING
 
 #ifdef RS_AUTOLOGIN
 	as      >> option('a',"auto-login"       ,conf.autoLogin      ,"AutoLogin (Windows Only) + StartMinimised");
 #endif // ifdef RS_AUTOLOGIN
-
+    as 		>> help('h',"help",QObject::tr("Display this help").toStdString().c_str());
+
+    if(as.helpRequested())
+    {
+        RsInfo() << "\n" <<
+            "+================================================================+\n"
+            "|     o---o                                             o        |\n"
+            "|      \\ /               - Retroshare GUI -            / \\       |\n"
```

Comment:
Does it intended to be misaligned (on the right side of the icon), or is it a stylish style?